var/spack/repos/builtin/packages/aocc/package.py

```diff
@@ -138,6 +138,7 @@ def cfg_files(self):
     }
 
     implicit_rpath_libs = ["libclang"]
+    stdcxx_libs = ("-lstdc++",)
```

Comment:
Good work here forcing this as a tuple. I was caught by this earlier by incorrectly using a string which cannot be appended to a list in application recipes.